_data/projects/particle/particle.yml

```diff
@@ -7,4 +7,4 @@ url: https://github.com/scikit-hep/particle
 # repo: only if different from url
 badges:
   pypi: Particle
-
+  conda-forge: Particle
```

Comment:
I'd make both of these lower case, but it shouldn't matter, I think

Reply:
Ah, I actually capitalised to match the PyPI tag. Will now uncapitalise both.